src/middlewares/task.middleware.js

```diff
@@ -93,7 +98,10 @@ export default (store: any) => (next: any) => (action: any) => {
           });
 
           child.on('exit', code => {
-            const wasSuccessful = code === 0 || code === null;
+            // For Windows Support
+            // Windows sends code 1 (I guess its because we foce kill??)
+            const successfulCode = isWin() ? 1 : 0;
```

Comment:
Is this really needed? I think this is working but is Windows returning 1 for killed tasks? I couldn't find anything in the docs. Are there any drawbacks if we're leaving this like it is?

I've also added the braces to `isWin` so that it is working as expected.

Reply:
I remember there was discussion about this but I forget where it landed. If this change isn't needed, that'd be great; I'm worried if Windows has exit code 1 on killed tasks.